tests/reference/tokens-docstring1-1355fbb.stdout

```diff
@@ -4,38 +4,32 @@
 (TOKEN ")") 10:10
 (TOKEN ":") 11:11
 (NEWLINE) 12:12
-(TOKEN "indent") 13:16
```

Comment:
The removal of this line and other subsequent lines caused the CI to fail.

Reply:
Actually, I didn't change them manually. I just ran `./run_tests.py -u`.

Reply:
Yup, No issues. I think now the CI should pass